dotCMS/src/main/java/com/dotmarketing/cmis/proxy/DotInvocationHandler.java

```diff
@@ -17,7 +17,10 @@
 import com.dotmarketing.util.WebKeys;
 import com.liferay.portal.model.User;
 
-//http://jira.dotmarketing.net/browse/DOTCMS-3392
+/**
+* @deprecated CMIS is deprecated and slated for removal.
+*/
+@Deprecated
 public class DotInvocationHandler implements InvocationHandler{

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)